x-pack/plugins/endpoint/common/generate_data.test.ts

```diff
@@ -3,7 +3,7 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
-import { EndpointDocGenerator, Event } from './generate_data';
+import { EndpointDocGenerator, Event, Tree, TreeNode } from './generate_data';
```

Comment:
:information_source: Just FYI, @kqualters-elastic and I found that the DocGenerator was updated recently to contain events with "category" fields that aren't documented anywhere (i.e. "library"). Not sure that matters here (and maybe it's even helfpul for testing?) just want to flag it for you.

Reply:
Thanks for pointing that out Brent. We must have missed that. I don't see `library` in the ecs docs either.